index.html

```diff
@@ -11,47 +11,107 @@
+               </div>
+               <div class="form-group pb-3 w-50">
+                  <label for="result">Result:</label>
+                  <p class="form-control" id="result">0</p>
+               </div> 
+            </div>
+         </form>
+      </div>
+
+
+      
+      <div class="section_search">
+         <h2 class="subtitle">Search rates</h2>
+         <form class="form p-4 formSearchRate--js">
+            <div class="d-flex flex-row justify-content-center align-items-center gap-3">
+               <div class="form-group pb-3 w-50">
+                  <label for="search-currency_from">From:</label>
+                  <input class="form-control" id="search-currency_from" type="text" placeholder="Enter base currency" required>
+               </div>
+               <div class="form-group pb-3 w-50">
+                  <label for="search-currency_to">To:</label>
+                  <input class="form-control" id="search-currency_to" type="text" placeholder="Enter target currency" required>
+               </div>
+            </div>
+            <button type="submit" class="btn btn-primary mb-3">Search Rate</button>
+            <div class="form-group pb-3">
+               <p class="form-control" id="search-result">Search result here ...</p>
+            </div>
+         </form>
+      </div>
+
+      <div class="section_rates_table">
+         <h2 class="subtitle">Exchange Rates Table</h2>
+         <table class="table table-striped">
+            <caption>Exchange Rates Table</caption>
+            <thead class="thead-light">
+              <tr class="table-col-currency">
+                <th scope="col">FROM \ TO</th>
+              </tr>
+            </thead>
+            <tbody class="table-row-currency">
+            </tbody>
+          </table>
+      </div>
+
+      <div class="section_create_rate">
          <div>
-            <h2 class="subtitle">Insert a new currency rate and add a new rate</h2>
-            <form class="form p-4 formRate--js">
+            <h2 class="subtitle">Insert a new currency rate</h2>
+            <form class="form p-4 formRate--js">            
                <div class="form-group pb-3">
                   <label for="currency_from">From:</label>
-                  <input class="form-control" id="currency_from" type="text" name="currency_from" placeholder="EUR" required>
+                  <input class="form-control currencyName_from" id="currency_from" type="text" name="currency_from" placeholder="EUR" required>
                </div>
                <div class="form-group pb-3">
                   <label for="currency_to">To:</label>
-                  <input class="form-control" id="currency_to" type="text" name="currency_to" placeholder="USD" required>
+                  <input class="form-control currencyName_to" id="currency_to" type="text" name="currency_to" placeholder="USD" required>
```

Comment:
Both `currencyName_from` and `currencyName_to` update naming convention and add `text-transform: uppercase;` to these elements so they are by default capitalised.